notebooks/01-visualize-amounts-bar-plots.ipynb

```diff
@@ -0,0 +1,1384 @@
+{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        ],  # tuple of RGB colors representing the stacked bars
</pre><p>Maybe convert the tuples to hex-string format? e.g. <code>"#ae4512"</code> most of the examples in the docs use hex strings and I think it would read a little more compactly. </p><p> </p><p>I also think "colors for each stack of bars" would be an improved comment</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>